server/src/main/java/school/journal/entity/Mark.java

```diff
@@ -1,11 +1,13 @@
 package school.journal.entity;
 
+import school.journal.entity.util.MarkType;
+
 import java.sql.Date;
 
 public class Mark {
     private Integer markId;
     private Integer value;
-    private Short type;
+    private String type;
```

Comment:
I've tried it, but when we try to access to it via getter from deserializer, was a trouble, so that is crutch.